src/one_wire.toit

```diff
@@ -222,17 +227,17 @@ class Protocol:
   Sends a reset to the receiver and reads whether a receiver is present.
   */
   reset -> bool:
-    old_threshold := rx_channel_.idle_threshold
-    rx_channel_.idle_threshold = RESET_IDLE_THRESHOLD_
+    old_threshold := channel_.idle_threshold
+    channel_.idle_threshold = RESET_IDLE_THRESHOLD_
     periods := [
       RESET_LOW_,
       RESET_HIGH_,
     ]
     try:
-      received_signals := rmt.transmit_and_receive --rx=rx_channel_ --tx=tx_channel_
-          --transmit=rmt.Signals 0
-          --receive=rmt.Signals.alternating --first_level=0 periods
-          4 * rmt.BYTES_PER_SIGNAL
+      received_signals := channel_.write_and_read
+          --during_read=rmt.Signals.alternating --first_level=0 periods
```

Comment:
I'd put --first_level on its own line.

Reply:
The `--first_level` is an argument to `rmt.Signals.alternating`.
Moved the construction of the signals to someplace earlier.